medusa/clients/nzb/nzbget.py

```diff
@@ -145,7 +145,8 @@ def sendNZB(nzb, proper=False):
                     if not nzb.provider.login():
                         return False
 
-                    data = nzb.provider.get_url(nzb.url, returns='content')
+                    # TODO: Check if this needs exception handling
+                    data = nzb.provider.session(nzb.url).content
```

Comment:
This is wrong for several reasons.  First the `.get` for the call is missing.  Secondly we need to create a session specifically for `nzbget`.  Additionally, if the response is needed in the future for further error handling, it is no longer provided.  A better approach would be:
```python
response = self.session.get(nzb.url)
data = response.content
```